<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>RaptorPlusPage</class>
    <widget class="QWidget" name="RaptorPlusPage">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>962</width>
                <height>629</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Form</string>
        </property>
        <layout class="QVBoxLayout" name="_PlusPageLayout">
            <property name="spacing">
                <number>0</number>
            </property>
            <property name="leftMargin">
                <number>0</number>
            </property>
            <property name="topMargin">
                <number>0</number>
            </property>
            <property name="rightMargin">
                <number>0</number>
            </property>
            <property name="bottomMargin">
                <number>0</number>
            </property>
            <item>
                <widget class="QWidget" name="_Header" native="true">
                    <property name="minimumSize">
                        <size>
                            <width>0</width>
                            <height>50</height>
                        </size>
                    </property>
                    <property name="maximumSize">
                        <size>
                            <width>16777215</width>
                            <height>50</height>
                        </size>
                    </property>
                    <property name="styleSheet">
                        <string notr="true"/>
                    </property>
                    <layout class="QHBoxLayout" name="_HeaderLayout">
                        <property name="spacing">
                            <number>12</number>
                        </property>
                        <property name="leftMargin">
                            <number>0</number>
                        </property>
                        <property name="topMargin">
                            <number>0</number>
                        </property>
                        <property name="rightMargin">
                            <number>0</number>
                        </property>
                        <property name="bottomMargin">
                            <number>0</number>
                        </property>
                        <item>
                            <widget class="QWidget" name="_HeaderPanel" native="true">
                                <layout class="QVBoxLayout" name="_HeaderPanelLayout">
                                    <property name="leftMargin">
                                        <number>0</number>
                                    </property>
                                    <property name="topMargin">
                                        <number>0</number>
                                    </property>
                                    <property name="rightMargin">
                                        <number>0</number>
                                    </property>
                                    <property name="bottomMargin">
                                        <number>0</number>
                                    </property>
                                    <item>
                                        <widget class="QLabel" name="_Title">
                                            <property name="minimumSize">
                                                <size>
                                                    <width>0</width>
                                                    <height>32</height>
                                                </size>
                                            </property>
                                            <property name="maximumSize">
                                                <size>
                                                    <width>16777215</width>
                                                    <height>32</height>
                                                </size>
                                            </property>
                                            <property name="styleSheet">
                                                <string notr="true"/>
                                            </property>
                                            <property name="text">
                                                <string/>
                                            </property>
                                            <property name="alignment">
                                                <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                                            </property>
                                        </widget>
                                    </item>
                                    <item>
                                        <widget class="QLabel" name="_Description">
                                            <property name="minimumSize">
                                                <size>
                                                    <width>0</width>
                                                    <height>0</height>
                                                </size>
                                            </property>
                                            <property name="maximumSize">
                                                <size>
                                                    <width>16777215</width>
                                                    <height>16777215</height>
                                                </size>
                                            </property>
                                            <property name="styleSheet">
                                                <string notr="true"/>
                                            </property>
                                            <property name="text">
                                                <string/>
                                            </property>
                                            <property name="alignment">
                                                <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
                                            </property>
                                        </widget>
                                    </item>
                                </layout>
                            </widget>
                        </item>
                        <item>
                            <spacer name="_HeaderSpacer">
                                <property name="orientation">
                                    <enum>Qt::Horizontal</enum>
                                </property>
                                <property name="sizeHint" stdset="0">
                                    <size>
                                        <width>40</width>
                                        <height>20</height>
                                    </size>
                                </property>
                            </spacer>
                        </item>
                    </layout>
                </widget>
            </item>
            <item>
                <widget class="QWidget" name="_Control" native="true">
                    <property name="minimumSize">
                        <size>
                            <width>0</width>
                            <height>36</height>
                        </size>
                    </property>
                    <property name="maximumSize">
                        <size>
                            <width>16777215</width>
                            <height>36</height>
                        </size>
                    </property>
                    <property name="styleSheet">
                        <string notr="true"/>
                    </property>
                    <layout class="QHBoxLayout" name="_ControlLayout">
                        <property name="spacing">
                            <number>0</number>
                        </property>
                        <property name="leftMargin">
                            <number>0</number>
                        </property>
                        <property name="topMargin">
                            <number>0</number>
                        </property>
                        <property name="rightMargin">
                            <number>0</number>
                        </property>
                        <property name="bottomMargin">
                            <number>0</number>
                        </property>
                        <item>
                            <widget class="QWidget" name="_TabPanel" native="true">
                                <property name="sizePolicy">
                                    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                                        <horstretch>0</horstretch>
                                        <verstretch>0</verstretch>
                                    </sizepolicy>
                                </property>
                                <property name="styleSheet">
                                    <string notr="true"/>
                                </property>
                                <layout class="QHBoxLayout" name="_TabPanelLayout">
                                    <property name="spacing">
                                        <number>0</number>
                                    </property>
                                    <property name="leftMargin">
                                        <number>0</number>
                                    </property>
                                    <property name="topMargin">
                                        <number>0</number>
                                    </property>
                                    <property name="rightMargin">
                                        <number>0</number>
                                    </property>
                                    <property name="bottomMargin">
                                        <number>0</number>
                                    </property>
                                    <item>
                                        <widget class="QPushButton" name="_TabPrev">
                                            <property name="sizePolicy">
                                                <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                                                    <horstretch>0</horstretch>
                                                    <verstretch>0</verstretch>
                                                </sizepolicy>
                                            </property>
                                            <property name="minimumSize">
                                                <size>
                                                    <width>30</width>
                                                    <height>0</height>
                                                </size>
                                            </property>
                                            <property name="maximumSize">
                                                <size>
                                                    <width>30</width>
                                                    <height>16777215</height>
                                                </size>
                                            </property>
                                            <property name="styleSheet">
                                                <string notr="true"/>
                                            </property>
                                            <property name="text">
                                                <string/>
                                            </property>
                                        </widget>
                                    </item>
                                    <item>
                                        <widget class="QPushButton" name="_TabCopy">
                                            <property name="sizePolicy">
                                                <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                                                    <horstretch>0</horstretch>
                                                    <verstretch>0</verstretch>
                                                </sizepolicy>
                                            </property>
                                            <property name="styleSheet">
                                                <string notr="true"/>
                                            </property>
                                            <property name="text">
                                                <string/>
                                            </property>
                                            <property name="checkable">
                                                <bool>true</bool>
                                            </property>
                                        </widget>
                                    </item>
                                    <item>
                                        <widget class="QPushButton" name="_TabNext">
                                            <property name="sizePolicy">
                                                <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                                                    <horstretch>0</horstretch>
                                                    <verstretch>0</verstretch>
                                                </sizepolicy>
                                            </property>
                                            <property name="minimumSize">
                                                <size>
                                                    <width>30</width>
                                                    <height>0</height>
                                                </size>
                                            </property>
                                            <property name="maximumSize">
                                                <size>
                                                    <width>30</width>
                                                    <height>16777215</height>
                                                </size>
                                            </property>
                                            <property name="styleSheet">
                                                <string notr="true"/>
                                            </property>
                                            <property name="text">
                                                <string/>
                                            </property>
                                        </widget>
                                    </item>
                                </layout>
                            </widget>
                        </item>
                        <item>
                            <spacer name="_ControlSpacer">
                                <property name="orientation">
                                    <enum>Qt::Horizontal</enum>
                                </property>
                                <property name="sizeHint" stdset="0">
                                    <size>
                                        <width>40</width>
                                        <height>20</height>
                                    </size>
                                </property>
                            </spacer>
                        </item>
                        <item>
                            <spacer name="_ControlSpaces">
                                <property name="orientation">
                                    <enum>Qt::Horizontal</enum>
                                </property>
                                <property name="sizeHint" stdset="0">
                                    <size>
                                        <width>40</width>
                                        <height>20</height>
                                    </size>
                                </property>
                            </spacer>
                        </item>
                        <item>
                            <widget class="QWidget" name="_ControlPanel" native="true">
                                <layout class="QHBoxLayout" name="_ControlPanelLayout">
                                    <property name="spacing">
                                        <number>12</number>
                                    </property>
                                    <property name="leftMargin">
                                        <number>0</number>
                                    </property>
                                    <property name="topMargin">
                                        <number>0</number>
                                    </property>
                                    <property name="rightMargin">
                                        <number>0</number>
                                    </property>
                                    <property name="bottomMargin">
                                        <number>0</number>
                                    </property>
                                    <item>
                                        <widget class="QWidget" name="_SearchPanel" native="true">
                                            <property name="styleSheet">
                                                <string notr="true"/>
                                            </property>
                                            <layout class="QHBoxLayout" name="_SearchPanelLayout" stretch="0,0">
                                                <property name="spacing">
                                                    <number>0</number>
                                                </property>
                                                <property name="leftMargin">
                                                    <number>0</number>
                                                </property>
                                                <property name="topMargin">
                                                    <number>0</number>
                                                </property>
                                                <property name="rightMargin">
                                                    <number>0</number>
                                                </property>
                                                <property name="bottomMargin">
                                                    <number>0</number>
                                                </property>
                                                <item>
                                                    <widget class="QLineEdit" name="_SearchEdit">
                                                        <property name="sizePolicy">
                                                            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                                                                <horstretch>0</horstretch>
                                                                <verstretch>0</verstretch>
                                                            </sizepolicy>
                                                        </property>
                                                        <property name="minimumSize">
                                                            <size>
                                                                <width>0</width>
                                                                <height>26</height>
                                                            </size>
                                                        </property>
                                                        <property name="maximumSize">
                                                            <size>
                                                                <width>16777215</width>
                                                                <height>26</height>
                                                            </size>
                                                        </property>
                                                        <property name="styleSheet">
                                                            <string notr="true"/>
                                                        </property>
                                                    </widget>
                                                </item>
                                                <item>
                                                    <widget class="QPushButton" name="_Search">
                                                        <property name="sizePolicy">
                                                            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                                                                <horstretch>0</horstretch>
                                                                <verstretch>0</verstretch>
                                                            </sizepolicy>
                                                        </property>
                                                        <property name="minimumSize">
                                                            <size>
                                                                <width>28</width>
                                                                <height>26</height>
                                                            </size>
                                                        </property>
                                                        <property name="maximumSize">
                                                            <size>
                                                                <width>28</width>
                                                                <height>26</height>
                                                            </size>
                                                        </property>
                                                        <property name="styleSheet">
                                                            <string notr="true"/>
                                                        </property>
                                                        <property name="text">
                                                            <string/>
                                                        </property>
                                                        <property name="iconSize">
                                                            <size>
                                                                <width>20</width>
                                                                <height>20</height>
                                                            </size>
                                                        </property>
                                                    </widget>
                                                </item>
                                            </layout>
                                        </widget>
                                    </item>
                                    <item>
                                        <widget class="QPushButton" name="_Execute">
                                            <property name="minimumSize">
                                                <size>
                                                    <width>0</width>
                                                    <height>26</height>
                                                </size>
                                            </property>
                                            <property name="maximumSize">
                                                <size>
                                                    <width>16777215</width>
                                                    <height>26</height>
                                                </size>
                                            </property>
                                            <property name="text">
                                                <string/>
                                            </property>
                                        </widget>
                                    </item>
                                    <item>
                                        <widget class="QPushButton" name="_Cancel">
                                            <property name="minimumSize">
                                                <size>
                                                    <width>0</width>
                                                    <height>26</height>
                                                </size>
                                            </property>
                                            <property name="maximumSize">
                                                <size>
                                                    <width>16777215</width>
                                                    <height>26</height>
                                                </size>
                                            </property>
                                            <property name="text">
                                                <string/>
                                            </property>
                                        </widget>
                                    </item>
                                </layout>
                            </widget>
                        </item>
                    </layout>
                </widget>
            </item>
            <item>
                <widget class="QStackedWidget" name="_Body">
                    <widget class="RaptorCopyPage" name="_CopyPage"/>
                </widget>
            </item>
        </layout>
    </widget>
    <customwidgets>
        <customwidget>
            <class>RaptorCopyPage</class>
            <extends>QWidget</extends>
            <header>../../../Src/Gui/Page/Plus/Copy/RaptorCopyPage.h</header>
            <container>1</container>
        </customwidget>
    </customwidgets>
    <resources/>
    <connections/>
</ui>
